src/examples/java/com/devcycle/examples/CloudExample.java

```diff
@@ -34,7 +33,7 @@ public static void main(String[] args) throws InterruptedException {
         Boolean variableValue = false;
         try {
             variableValue = client.variableValue(user, VARIABLE_KEY, defaultValue);
-        } catch(DevCycleException e) {
+        } catch (IllegalArgumentException e) {
```

Comment:
Is there any place we would actually want to keep a DVCException?

Reply:
That is a good question. We really only use it to wrap an HTTP error for a lot of the Cloud SDK methods. 

I think for operations which can potentially generate an error and be retried it makes sense, like tracking a single event. The Local SDK seems to swallow most of the exceptions and just log errors instead. It would be nice for both SDKs to do the same thing.

Reply:
I think i'll push further reworking of the exceptions to a separate PR, the Java SDK is kind of a mess of exception handling and even uses exceptions for flow control in a couple places which is gross.

Reply:
Yeah that's valid; as long as this state is still functional